### PR TITLE
Tailwind setup

### PR DESCRIPTION
**Pull request by LauraBLei**
-() Setup tailwind

### CODE REVIEW

File: node_modules/.bin/atob
Comment (Mithu225) [inline node_modules/.bin/atob:1]: Please DO NOT push "node_modules" to the feature branch. you should remove it before pushing and creating `.gitignore` and put node_modules to that file. and commit/push the `.gitignore`  file to your branch

https://www.w3schools.com/git/git_ignore.asp?remote=github

**Review by Whisperpiano — APPROVED** [review]
After the call I understood about the node modules, thank you Laura! :)

**Review by Whisperpiano — APPROVED** [review]

**Review by Whisperpiano — APPROVED** [review]

**Review by shiwa4656 — APPROVED** [review]

**Review by lem31 — APPROVED** [review]

**Comment by Whisperpiano** [comment]
Ready to merge here.

**Comment by lem31** [comment]
I noticed this was closed without merging, so I have reopened it and merged and closed it again :-)

**Pull request merged**